apps/server/src/modules/health/health-api.module.ts

```diff
@@ -0,0 +1,12 @@
+import { Module } from '@nestjs/common';
+
+import { HealthController } from './controller';
+import { HealthcheckRepo } from './repo';
```

Comment:
```suggestion
import { HealthCheckRepo } from './repo';
```

Reply:
Done in https://github.com/hpi-schul-cloud/schulcloud-server/pull/4191/commits/9f7cb45820e06668af950204e05f82983c112e54 .